tutorials/javaFxPart4.md

```diff
@@ -109,6 +109,28 @@ We will get to that later.
 
    ![Settings for DialogBox](images/javafx/DialogBoxController.png)
 
+<box type="info" seamless>
+
+  Using the FXML files above, text overrun may occur when the text to be displayed exceeds the size of the label, causing the text to not be fully displayed and ending with `...` instead.
+
+  To fix this issue, we can implement one of the suggested changes below to the FXML files:
+
+  <b>1. Changing MainWindow.fxml</b>
+  
+  Remove `prefHeight` for `VBox`:
+  ```xml
+  <VBox fx:id="dialogContainer" prefWidth="388.0" />
+  ```
+
+  <b>2. Changing DialogBox.fxml</b>
+  
+  Set `minHeight` for `Label`:
+  ```xml
+  <Label fx:id="dialog" text="Label" wrapText="true" minHeight="-Infinity"/>
+```
```

Comment:
For this change, an explanation along the lines of:

```markdown
This setting allows the label to have no minimum height constraint, meaning it can shrink or expand vertically as much as needed. This flexibility ensures that if the text wrapped within the label requires more height due to wrapping, the label can expand to fit this text without truncating it. This change is particularly useful in ensuring that all content of the label is visible, regardless of the amount of text.
```

Would be fitting.

Reply:
Thank you for the suggestion, this issue has been resolved in PR #44